platform/src/layouts/root_layout/side_bar.rs

```diff
@@ -20,6 +20,16 @@ pub struct SidebarProps {
 #[component]
 pub fn SideBar(props: SidebarProps) -> Element {
     let mut api: OrganizationApi = use_context();
+
+    let _ = use_resource(move || {
```

Comment:
```suggestion
    let _ = use_server_future(move || {
```
이거 안쓰고 use_resource 를 쓰는 이유가 있을까요?

Reply:
use_server_future로 사용하니까 데이터를 못 불러오더라구요.
일단 use_resource로 가다가 use_server_future로 추후에 리펙토링해보도록 하겠습니다!